### ISSUE TITLE
incompatible number of blobs for layer bn0

### ISSUE THREAD

**Issue by rainmakerMa** [cypw/MXNet2Caffe#7]
hi,I am trying to convert a model trained by mxnet into caffemodel,however after i get the caffemodel file,and tring to test the model on caffe framework,the error occurs:check failed:target_blobs.size() == source_layer.blobs_size() (3 vs 5) incompatible number of blobs for layer bn0.how to solve it

**Comment by suntingtencent** [comment]
Has this issue being solved ??? Pls kindly reply

**Comment by 77h2l** [comment]
> Has this issue being solved ??? Pls kindly reply

yeah,the implementation of Batchnorm layer of different versions of caffe matters,you need to check your caffe's version,for example,Nvidia's caffe implement the BN and Scale together
